stairctl/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
 
     // JSON processing
     ext {
-        jackson = '2.16.1'
+        jackson = '2.16.2'
```

Comment:
I updated Jackson dependencies in https://github.com/DataBiosphere/stairway/pull/132, put apparently another patch release just landed.